fe/be-java-extensions/java-udf/src/main/java/org/apache/doris/udf/UdfExecutor.java

```diff
@@ -140,26 +142,91 @@ private Method findPrepareMethod(Method[] methods) {
         return null; // Method not found
     }
 
-    public ClassLoader getClassLoader(String jarPath, String signature, long expirationTime)
-            throws MalformedURLException, FileNotFoundException {
-        ClassLoader loader = null;
-        if (jarPath == null) {
-            // for test
-            loader = ClassLoader.getSystemClassLoader();
-        } else {
+    public UdfClassCache getClassCache(String className, String jarPath, String signature, long expirationTime,
+            Type funcRetType, Type... parameterTypes)
```

Comment:
I check the failed two test case, not give jarPath in create function.
so pass jarPath = null to BE, 
and maybe should give the path to custom_lib?